tpp-private/private.php

```diff
@@ -18,148 +18,139 @@
 </head>
 
 <body>
-<div class="container">
+    <div class="container">
 
-    <header>
-        <h1>Placeholder for actual header</h1>
-    </header>
+        <header>
+            <h1>Placeholder for actual header</h1>
+        </header>
 
+        <div class="profileImage">
+            <img src="profileimg/img0.gif" alt="" width=200 height=200 />
+        </div>
 
-    <div class="profileImage">
-        <img src="profileimg/img0.gif" alt="" width=200 height=200/>
-    </div>
-
-    <div class="talentInfo">
-        <p>Insert talent fname/lname</p>
-        <p><b>Specialities: </b>Guitar, Singer, Songwriter, Actor </p>
-    </div>
-    <div class="email">
-        <p>Email address: <a href="mailto:testemail@adus.com">testemail@adus.com</a></p>
-    </div>
-
-
-    <div class="phototitle">
-        <h1><b>Photos</b></h1>
-    </div>
-
+        <div class="talentInfo">
+            <p>Insert talent fname/lname</p>
+            <p><b>Specialities: </b>Guitar, Singer, Songwriter, Actor </p>
+        </div>
+        <div class="email">
+            <p>Email address: <a href="mailto:testemail@adus.com">testemail@adus.com</a></p>
+        </div>
 
-    <div class="photoholder">
+        <div class="phototitle">
+            <h1><b>Photos</b></h1>
+        </div>
 
 
-        <!--  first img should be this one for talents so they can add more images later on to the project  -->
-        <div class="addimg">
-            <form action="#" method="post" enctype="multipart/form-data">
-                <label>
-                    <input type="file" name="uploadImg" id="" style="display:none">
-                    <img src="stockphotoholder/addimg.png" alt="" height="200" width="200">
-                    <figcaption>
-                        <p>Add a new image</p>
-                    </figcaption>
-                </label>
-        </div>
-        <!--  <div class="addimg">
+        <div class="photoholder">
+
+            <!--  first img should be this one for talents so they can add more images later on to the project  -->
+            <div class="addimg">
+                <form action="#" method="post" enctype="multipart/form-data">
+                    <label>
+                        <input type="file" name="uploadImg" id="" style="display:none">
+                        <img src="stockphotoholder/addimg.png" alt="" height="200" width="200">
```

Comment:
same don't think we should have inline style. let's consult Paul about this as well